Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -102,6 +102,22 @@ public extension URL {
         }
     }
 
+    /// SwifterSwift: Generates new URL that does not have scheme.
+    ///
+    ///        let url = URL(string: "https://domain.com")!
+    ///        print(url.droppedScheme()) // prints "domain.com"
+    public func droppedScheme() -> URL? {
+        // https://github.com/SwifterSwift/SwifterSwift/issues/464
```

Comment:
We don't need to put a reference to the issue here

Reply:
@SD10 I've removed the reference.